Source/WebCore/html/HTMLMarqueeElement.cpp

```diff
@@ -157,24 +157,24 @@ void HTMLMarqueeElement::stop()
 
 unsigned HTMLMarqueeElement::scrollAmount() const
 {
-    return limitToOnlyHTMLNonNegative(attributeWithoutSynchronization(scrollamountAttr), RenderStyle::initialMarqueeIncrement().intValue());
+    return limitToOnlyHTMLNonNegative(attributeWithoutSynchronization(scrollamountAttr), 6);
 }
-    
+
 void HTMLMarqueeElement::setScrollAmount(unsigned scrollAmount)
 {
-    setUnsignedIntegralAttribute(scrollamountAttr, limitToOnlyHTMLNonNegative(scrollAmount, RenderStyle::initialMarqueeIncrement().intValue()));
+    setUnsignedIntegralAttribute(scrollamountAttr, limitToOnlyHTMLNonNegative(scrollAmount, 6));
```

Comment:
Surprised to see a literal number here rather than a named constant. Often we host such values to the top of the file?